__test__/generateDescription.test.js

```diff
@@ -0,0 +1,98 @@
+const generateDescription = (characteristic, rating) => {
```

Comment:
I tried importing this function directly, but jest was somehow not playing nicely with babel, so import/export statements were a no-go. Something to take a look at in SDC, perhaps.